### PR TITLE
Fixed ground primitives in orthographic mode

### LINKED ISSUE #5110: GroundPrimitives are broken in orthographic projection
Most likely a clipping or depth clamping issue.

See https://github.com/AnalyticalGraphicsInc/cesium/pull/5021#issuecomment-281544461

### PR DESCRIPTION
**Pull request by IanLilleyT**
Fixes #5110 

There were two problems going on:
1) `czm_geometricToleranceOverMeter` was `NaN` because it assumed the frustum had an `fov` property, which is not the case for orthographic. This made shadow volumes extend REALLY far which caused some strange behavior, likely floating point precision related.
2) The shadow volume geometry behind the near plane was being clipped, confusing the shadow volume algorithm into thinking the mesh was not watertight, which caused it to be shaded in weird ways. So now it clamps to the far AND the near plane. Not sure why this problem doesn't show up for perspective + multi-frustum. It's possible I haven't found the perfect camera angle to trigger it.

Before:

Screen Shot 2020-05-14 at 11 22 33 PM

After:

Screen Shot 2020-05-14 at 11 22 45 PM

Before:

Screen Shot 2020-05-14 at 11 26 43 PM

After:

Screen Shot 2020-05-14 at 11 27 14 PM

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @IanLilleyT!
* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

**Comment by lilleyse** [comment]
> The shadow volume geometry behind the near plane was being clipped, confusing the shadow volume algorithm into thinking the mesh was not watertight, which caused it to be shaded in weird ways. So now it clamps to the far AND the near plane. Not sure why this problem doesn't show up for perspective + multi-frustum. It's possible I haven't found the perfect camera angle to trigger it.

I thought this was what the stencil preload pass was for. Is the stencil preload pass even needed anymore?

**Comment by lilleyse** [comment]
> I thought this was what the stencil preload pass was for. Is the stencil preload pass even needed anymore?

Actually we should just scrap the three-pass shadow volume technique. I have the two-pass approach working on a branch and will open a PR after this is merged. The near/far clamping here is still important.

File: Source/Scene/Primitive.js [hunk 1]
```diff
@@ -1136,13 +1136,9 @@ function depthClampVS(vertexShaderSource) {
   // It is not to workaround an issue with Edge:
   //     https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/12120362/
```
Comment (lilleyse) [inline Source/Scene/Primitive.js:1137]: Is this comment still relevant?

Comment (IanLilleyT) [reply Source/Scene/Primitive.js]: nope, the link is dead and edge is based on chromium now

Comment (IanLilleyT) [reply Source/Scene/Primitive.js]: nope, removed.

**Pull request merged**